tests/Auth/AuthDatabaseTokenRepositoryTest.php

```diff
@@ -13,19 +13,11 @@
 
 class AuthDatabaseTokenRepositoryTest extends TestCase
 {
```

Comment:
We never need to pretent that Carbon::now is different than now in this test class